xml/ay_custom_user_scripts.xml

```diff
@@ -34,7 +34,7 @@
  </para>
 
  <para>
-  All scripts need to be in the &lt;scripts&gt; section.
+   Except for <literal>init scripts</literal>, all scripts must be included in the <scripts> section.
```

Comment:
This breaks, I did not look closely enough. `scripts` needs to be escaped as in the original (`&lt;scripts&gt`)